plugins/autochop.cpp

```diff
@@ -309,6 +335,18 @@ static int do_chop_designation(bool chop, bool count_only, int *skipped = nullpt
             continue;
         }
 
+        trees_by_size.insert(pair<int, int>(estimate_logs(plant), i));
+    }
+
+    //designate
+    multimap<int, int>::iterator itr;
+    for (itr = trees_by_size.begin(); itr != trees_by_size.end(); itr++)
```

Comment:
You can simplify this to:
`for (auto & entry : trees_by_size)`
And then get the plant via:
`df::plant * plant = entry.second;`